src/lib/customTraceType.js

```diff
@@ -0,0 +1,32 @@
+export function plotlyToCustom(trace) {
```

Comment:
`plotlyToCustomTrace` and `customTraceToPlotly`? Not a big deal but it isn't apparent from the function name what it is actually doing.

Reply:
(could be custom axes or custom sandwiches :) )

Reply:
or even more explicit `plotlyTraceToCustomTrace` and `customTraceToPlotlyTrace` ? This ought to appeal to someone who likes typing `MultiTextEditor` instead of `TextEditor` ;)

Reply:
haha, alright!